x-pack/plugins/session_view/public/components/SessionViewDetailPanel/styles.ts

```diff
@@ -35,24 +35,18 @@ export const useStyles = ({ height = 500 }: StylesDeps) => {
       overflowY: 'auto',
       position: 'absolute',
       top: '8px',
-      right: '-100%',
+      right: '-424px',
     };
 
-    const detailPanelIn: Array<string | CSSObject> = [
-      slideIn.styles,
-      {
-        ...detailPanel,
-        animation: `${slideIn.name} 200ms ease forwards`,
-      },
-    ];
+    const detailPanelIn: CSSObject = {
+      ...detailPanel,
+      animation: `${slideIn} 200ms ease forwards`,
```

Comment:
Also, just found we can use animations from [EuiTheme](https://elastic.github.io/eui/#/theming/more-tokens) to keep them consistent across Kibana:

```js
    const detailPanelIn: CSSObject = {
      ...detailPanel,
      animation: `${slideIn} ${euiTheme.animation.normal} ease forwards`,
    };

    const detailPanelOut: CSSObject = {
      ...detailPanel,
      animation: `${slideOut} ${euiTheme.animation.fast} ease forwards`,
    };
```

entire code would be like:

```js
export const useStyles = ({ height = 500 }: StylesDeps) => {
  const { euiTheme } = useEuiTheme();

  const cached = useMemo(() => {
    const { animation } = euiTheme;

    const slideIn = keyframes({
      from: {
        right: '-424px',
      },
      to: {
        right: '0',
      },
    });

    const slideOut = keyframes({
      from: {
        right: '0',
      },
      to: {
        right: '-424px',
      },
    });

    const detailPanel: CSSObject = {
      width: '424px',
      height: `${height}px`,
      overflowY: 'auto',
      position: 'absolute',
      top: '8px',
      right: '-424px',
    };

    const detailPanelIn: CSSObject = {
      ...detailPanel,
      animation: `${slideIn} ${animation.normal} ease forwards`,
    };

    const detailPanelOut: CSSObject = {
      ...detailPanel,
      animation: `${slideOut} ${animation.fast} ease forwards`,
    };

    return {
      detailPanelIn,
      detailPanelOut,
    };
  }, [height, euiTheme]);

  return cached;
};
```

Reply:
oh cool! will update :)